pkgs/development/r-modules/generate-shell.nix

```diff
@@ -5,13 +5,16 @@ stdenv.mkDerivation {
 
   buildCommand = "exit 1";
 
-  buildInputs = [ wget ];
+  buildInputs = [ wget
+                cacert
+                nix];
```

Comment:
The spacing here looks a bit funny. Two options which are consistent with the rest of nixpkgs are:

```
  buildInputs = [
    wget
    cacert
    nix
  ];
  ```
or
```
  buildInputs = [ wget cacert nix ];
```

Reply:
Thank you for the feedback, fixed and pushed.